Traceback (most recent call last):
  File "C:/Users/lk3/source/repos/Python_Grundkurs_2024_09_09/M012.py", line 56, in <module>
    f.beschleunige(500)
  File "C:/Users/lk3/source/repos/Python_Grundkurs_2024_09_09/M012.py", line 48, in beschleunige
    raise ValueError(f"Geschwindigkeit darf {self.maxV}km/h nicht überschreiten")
ValueError: Geschwindigkeit darf 200km/h nicht überschreiten
